crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -181,7 +203,77 @@ pub fn execute_call_entry_point(
     // endregion
 }
 
-fn remove_syscall_resources_and_exit_success_call(
+fn evaluate_execution_result(
+    result: ContractClassEntryPointExecutionResult,
+    call: ExecutableCallEntryPoint,
+    current_tracked_resource: TrackedResource,
+    is_revertable: bool,
+) -> ContractClassEntryPointExecutionResult {
+    match result {
+        Ok(res) => {
+            // TODO: It should check `context.versioned_constants().enable_reverts` instead of `is_revertable`
+            if res.call_info.execution.failed && !is_revertable {
+                return Err(EntryPointExecutionErrorWithTrace {
+                    source: EntryPointExecutionError::ExecutionFailed {
+                        error_trace: extract_trailing_cairo1_revert_trace(
+                            &res.call_info,
+                            Cairo1RevertHeader::Execution,
+                        ),
+                    },
+                    trace: res.vm_trace,
+                });
```

Comment:
nit: Reduced nestings 😅 
```suggestion
                let error_trace = extract_trailing_cairo1_revert_trace(
                    &res.call_info,
                    Cairo1RevertHeader::Execution,
                );
                let source = EntryPointExecutionError::ExecutionFailed { error_trace };
                return Err(EntryPointExecutionErrorWithTrace {
                    source,
                    trace: res.vm_trace,
                });
```

Reply:
The same comment as above